src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -385,7 +385,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <ItemGroup>
       <_ResolvedCopyLocalPublishAssets Include="@(ReferenceCopyLocalPaths)"
-                                       Exclude="@(_ResolvedCopyLocalPublishAssets)"
+                                       Exclude="@(_ResolvedCopyLocalBuildAssets)"
```

Comment:
Explanation of change: `ReferenceCopyLocalPaths` is a superset of `_ResolvedCopyLocalBuildAssets` when we copy local package assets for build.  The intention of this was to add the subset of copy local paths that were not copied local from the build assets (`_ResolvedCopyLocalPublishAssets` may be a different set of items that was resolved just for publish).  If we're reusing the build copy local assets, then `_ResolvedCopyLocalPublishAssets` will already contain `_ResolvedCopyLocalBuildAssets`.